check_whitespace.c

```diff
@@ -2,11 +2,11 @@
 #include <string.h>
 #include <stdlib.h>
 
-/*
- * Strips spaces from both the front and back of a string,
- * leaving any internal spaces alone.
- */
+
+ 
 char* strip(char* str) {
+  // Strips spaces from both the front and back of a string,
```

Comment:
Why did you move the comment into the function definition? It's not wrong, I'm just curious.